qemu/tests/rng_stress.py

```diff
@@ -40,7 +40,7 @@ def get_available_rng(session):
         verify_cmd = params["driver_available_cmd"]
         try:
             output = session.cmd_output_safe(verify_cmd)
-            rng_devices = re.findall(r"virtio_rng.\d+", output)
+            rng_devices = re.findall(r"virtio", output)
```

Comment:
Why not use parameter instead of fixed string.  `driver_name` can match it,  and this is the check method for `driver_verifier_cmd` in rng_bat.py.

Reply:
Updated, thanks!